scripts/build.ts

```diff
@@ -411,21 +416,7 @@ async function buildJS () {
 			outputOptions: {
 				...defaultOutputOptions,
 				dir: './dist/cjs',
-			},
-		},
-		global: {
-			rollupOptions: {
-				...defaultRollupOptions,
-				input: {
-					'prism': path.join(SRC_DIR, 'auto-start.ts'),
-				},
-			},
-			outputOptions: {
-				...defaultOutputOptions,
-				format: 'iife',
-				name: 'Prism',
-				exports: 'default',
-				extend: true,
+				chunkFileNames: '_chunks/[name]-[hash].js',
```

Comment:
Why are we using chunks, and why only in CJS?

Reply:
I removed chunks from all the bundles